src/app/(main)/stories/[id]/StoryDetailClient.tsx

```
@@ -0,0 +1,174 @@
+  const storyId = Number(id);
+
+  if (!id || isNaN(storyId)) {
+    return (
+      <div className="flex flex-col items-center justify-center min-h-[400px]">
+        <h2 className="text-2xl font-bold text-Gray-7 mb-4">잘못된 접근</h2>
+        <p className="text-Gray-5">올바른 경로로 접근해주세요.</p>
+      </div>
+    );
+  }
+
+  const { data: story, isLoading, isError, error } = useStoryDetailQuery(storyId);
```

ESLint

[error] React Hook "useStoryDetailQuery" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)